specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/examples/NetworkManagerSecurityConfigurationEvaluateImport.json

```diff
@@ -0,0 +1,124 @@
+{
```

Comment:
This API is not future proof? What if we want add new imports like import from FW or something else?
We are restricitng it to NSG.